packages/api/src/controllers/stream.test.ts

```diff
@@ -712,6 +713,28 @@ describe("controllers/stream", () => {
         const document = await db.stream.get(stream.id);
         expect(db.stream.addDefaultFields(document)).toEqual(updatedStream);
       });
+      it("should extract region from redirected playback url", async () => {
```

Comment:
It might be good to have some flv urls in here to verify nothing breaks: 
e.g `/flv/<playback-id>`

Reply:
As pointed out below, we don't use it for `/flv`